pocs/tests/test_pocs.py

```diff
@@ -324,6 +324,38 @@ def test_unsafe_park(pocs):
     pocs.power_down()
 
 
+def test_no_ac_power(pocs):
+    # Simulator makes AC power safe
+    assert pocs.has_ac_power() is True
+
+    # Remove 'power' from simulator
+    pocs.config['simulator'] = ['camera', 'mount', 'weather']
```

Comment:
Why not `pocs.config['simulator'].remove('power')`?

Reply:
Yes, that's much better. Thanks!